climpred/bootstrap.py

```diff
@@ -234,6 +236,108 @@ def create_pseudo_members(control):
     )
 
 
+def bootstrap_uninit_pm_ensemble_from_control_cftime(init_pm, control):
+    """
+    Create a pseudo-ensemble from control run. Bootstrap random numbers for years to
```

Comment:
I think the first sentence should move up to be the one line description. E.g. "Create a pseudo-ensemble by bootstrapping a control run." Then the rest goes down below that.